.changeset/chore-fix-remaining-unit-test-errors.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-docs': patch
+---
+
+## Chore: Fix remaining unit test errors
```

Comment:
Because this is something our adopters _could_ care about (particularly the warnings), we do want this to be part of the `dom` changes. Also, we don't want to use `##` in the message because it will mess up the formatting in the docs [changelog](https://react-magma.cengage.com/version/4.7.0/api-intro/changelog/)
```suggestion
'react-magma-dom': patch
---

chore: Fix unit test errors and warnings
```